node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -251,7 +251,7 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
       opt[Int](descr = "Maximum size of message that can be sent via transport layer")
 
     val threadPoolSize =
-      opt[Int](descr = "Maximum number of threads used by rnode")
+      opt[Int](descr = "Maximum number of threads used by rnode", hidden = true)
```

Comment:
If `hidden = true` hides this flag from the help output why do we need this option in the first place?

Reply:
I want to remove this flag in the future. But ATM I'm just hiding it to not break some existing scripts. This flag is like _depricated_.